operator/scripts/utils/yaml.go

```diff
@@ -66,11 +66,16 @@ func SplitYAML(yamlBytes []byte) ([][]byte, error) {
 	r := bytes.NewReader(yamlBytes)
 	dec := goyaml.NewDecoder(r)
 	results := make([][]byte, 0)
-	var value map[string]interface{}
-	for eof := dec.Decode(&value); errors.Is(eof, io.EOF); eof = dec.Decode(&value) {
```

Comment:
oh it was a double Decode here that's the issue, right ?

Reply:
It should be `...; !errors.Is(...); ...` the negation was missing

But yeah, the code is too clever for me :-)